python/tvm/contrib/hexagon/session.py

```diff
@@ -60,10 +60,10 @@ def __init__(
         rpc_receive_buffer_size_bytes: int = 2 * 1024 * 1024,
     ):
         self._launcher = launcher
-        self._session_name = session_name
-        self._remote_stack_size_bytes = remote_stack_size_bytes
-        self._rpc_receive_buffer_size_bytes = rpc_receive_buffer_size_bytes
-        self._remote_kw = remote_kw
+        self._session_name: str = session_name
```

Comment:
can you update the docstring to describe the range of acceptable values and the purpose?

Reply:
done.

Reply:
I think it's probably also helpful to have a comment in the docstring about why there are two variants. e.g. running with hexagon as an llvm subtarget

Reply:
added.